src/libs/tmdk/package.json

```diff
@@ -7,8 +7,8 @@
     "tmdk": "./build/cli.js"
   },
   "scripts": {
-    "test": "./node_modules/.bin/jest",
-    "integ-test": "./node_modules/.bin/jest -c jest.config.integration.js",
+    "test": "./node_modules/.bin/jest --silent",
+    "integ-test": "./node_modules/.bin/jest --silent -c jest.config.integration.js",
```

Comment:
(just curious)

IIUC `--silent` will disable all the output logging right?

Is it a best practice to disable? I guess I'm wondering if is more useful to keep: if the tests fail, then the output may be useful to see what happened without having to re-run. If the tests pass, then would we just ignore the jest output anyway?

Reply:
Silent disables console logging but exceptions and fix recommendations are still printed when tests fail. Given the amount of logging in tmdk and the tests I think it would be better to silence them by default. If the exception given by the failed test is not enough to track down the issue then I think it is reasonable to expect the user to re-run with full logging. I see this as analogous to requiring` --debug` for finer grained logging when trying to track down a particular issue